pom.xml

```diff
@@ -31,7 +31,7 @@
     </distributionManagement>
 
     <properties>
-        <dependency.alfresco-core.version>6.20</dependency.alfresco-core.version>
+        <dependency.alfresco-core.version>jdk8-1</dependency.alfresco-core.version>
```

Comment:
When you release core, please use 6.21 (I'm assuming this is correct). If we need to change the 6.20 version for older versions, we can always go to 6.20.1.

Reply:
Yes, 6.21 will be the next release version.